zipkin-lens/src/components/Browser/BrowserHeader.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import BrowserHeader from './BrowserHeader';
+import { sortingMethods } from './sorting';
+
+describe('<BrowserHeader />', () => {
+  it('should render correct the number of results', () => {
```

Comment:
I am very happy you put in a "toehold test" like mentioned in @swankjesse's blog https://publicobject.com/2019/02/17/toehold-test/

this means later, we can test edge cases like zero results. Starting the first test class reduces burden of later change!